### PR TITLE
Bug 1333175, added automatically defined environment variables

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1333175

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@luciddreamz PTAL

 @adellape @bfallonf @tpoitras @tnguyen-rh  Please peer review

File: dev_guide/environment_variables.adoc [hunk 1]
```diff
@@ -121,6 +121,26 @@ Instead, it writes a YAML object with updated environment `STORAGE=/local` to ne
...
+.Automatically Added Environment Variables
+[cols="2",options="header"]
+|===
+
+|Variable Name |Description
+
+|`*<SVCNAME>_SERVICE_HOST*`
+|Service host variable automatically created by Kubernetes.
+
+|`*<SVCNAME>_SERVICE_POST*`
```
Comment (luciddreamz) [inline dev_guide/environment_variables.adoc]: This should be `*<SVCNAME>_SERVICE_PORT*` (port not post)

File: using_images/db_images/mysql.adoc [hunk 2]
```diff
@@ -140,6 +140,12 @@ application.
 |Optional password for the root user. If this is not set, then remote login to
 the root account is not possible. Local connections from within the container
 are always permitted without a password.
+
+|`*MYSQL_SERVICE_HOST*`
+|Service host variable automatically created by Kubernetes.
+
+|`*MYSQL_SERVICE_POST*`
```
Comment (luciddreamz) [inline using_images/db_images/mysql.adoc]: Service_Port not Service_Post here also

**Comment by tnguyen-rh** [comment]
@ahardin-rh I think the descriptions could be more descriptive.  Maybe i'm weird, but it seems obvious to me that they are variables created by Kubernetes, so why include that impl detail .  :frowning:  The rest of the description is the name of the variable "broken down", so it doesn't add too much interesting information.  As a reader, i would expect the description of a var to mention where it's used (and possibly when), and the valid values (number, symbol of a small set, etc) of the var.

**Comment by tnguyen-rh** [comment]
@ahardin-rh On second thought, my previous comment was made in a sleep-deprived state and thus are overly cranky.  Perhaps we can add a simple example that would clearly show what values are okay, w/o having a long-winded description?

**Comment by ahardin-rh** [comment]
@luciddreamz Thanks for catching that typo. That is fixed.

@tnguyen-rh I updated this with example usage in the Environment Variables section. Thoughts?

**Comment by tnguyen-rh** [comment]
@ahardin-rh LGTM! :+1:

**Comment by ahardin-rh** [comment]
@luciddreamz good to merge? Thanks!

**Comment by luciddreamz** [comment]
Looks good

On Thursday, June 23, 2016, Ashley Hardin <notifications@github.com> wrote:

> @luciddreamz <https://github.com/luciddreamz> good to merge? Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/2326#issuecomment-228128687>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA4qpcZJoXTgEgjwedjzORDqJrQLBT-Tks5qOsefgaJpZM4I7J6u>
> .
>


-- 
Jacob Lucky
jlucky@redhat.com | (919) 890-8540

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../dev_guide/environment_variables.html[Managing Environment Variables]
|Added information about automatically added environment variables.
%
|link:../using_images/db_images/mysql.html[Database Images -> MySQL]
|Added `*MYSQL_SERVICE_HOST*` and `*MYSQL_SERVICE_PORT*`to the list of MySQL Environment Variables.
%

**Pull request merged**